requirements.in

```diff
@@ -0,0 +1 @@
+flagsmith
```

Comment:
For the java sdk, I've been expecting that the provider code would live in a separate repository. We should discuss this as a broader group.

Reply:
Ya, I feel the same, I made this comment: https://github.com/open-feature/python-sdk/pull/1#discussion_r885898764